udfs/migration/redshift/interval_literal_to_seconds.sql

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION
+  rs.interval_literal_to_seconds(interval_literal STRING) AS ( /* For intervals in Redshift, there are 360 days in a year. */ (
+    SELECT
+      SUM(CASE
```

Comment:
Can we move CASE to the next line, so it is more readable?

```
SUM (
  CASE
    WHEN ... THEN
    ...
    ELSE ...
  END
) -- END OF  SUM
```

Reply:
What are the code guidelines? I used BQ UI to format the code. Is BQ provides formatting we may stick to it.

Reply:
For internal code submit, I encourage us to use go/sqlguide. For external users, they can only rely on BQ UI to format the code, however, sometimes, it doesn't emit the best readable code.

Reply:
I've been adhering to go/sql-style